certbot/main.py

```diff
@@ -388,6 +389,85 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _delete_if_appropriate(config): # pylint: disable=too-many-locals,too-many-branches
+    """Does the user want to delete their now-revoked certs?"""
+    display = zope.component.getUtility(interfaces.IDisplay)
+    reporter_util = zope.component.getUtility(interfaces.IReporter)
+
+    if config.namespace.noninteractive_mode:
```

Comment:
We can remove this check now that we have the mechanism at line 431; if we aren't sure what they mean, we don't delete, so it's ok to run non-interactively.